strategy-network-or-cache/README.md

```diff
@@ -0,0 +1,17 @@
+# Network or cache
+The recipe provides a service worker trying to retrieve the most updated content
```

Comment:
suggestion: "The recipe provides a service worker that tries to retrieve the most up to date content, with no detriment of user experience" (although I'm not sure what does the latter exactly mean-is it the interface is smooth? download is fast? it is downloading things intelligently? whatever it is, it would be better to see a more accurate description)